news/_posts/2019-07-04-1.6.1.x-what-s-next.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "1.6.1.x: what’s next"
+subtitle: "The end of 1.6 official support by the PrestaShop company. And the beginning of something new."
+date:   2019-07-04 11:30:00
+authors: [ AntoineThomas ]
+icon: icon-flag-corner
+tags:
+ - 1.6
+ - Maintenance
+---
+
+
+This last year, the PS company has been facing a dilemma: how to dedicate 100% of the developers’ time to move forward faster on the 1.7 version of the open source project, something everybody agrees on. And, at the same time, provide more time for merchants still using the 1.6 version to get ready to upgrade, by keeping them safe in case of critical issue [after the official support ends on the 30th of June 2019[(https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6). We have come up with a solution, thanks to the initiative and support of a few volunteers from the PrestaShop developer community: allowing them to take over the responsibility of maintaining PrestaShop 1.6.1.x beyond the end of official support.
```

Comment:
```suggestion
This last year, the PS company has been facing a dilemma: how to dedicate 100% of the developers’ time to move forward faster on the 1.7 version of the open source project, something everybody agrees on, and, at the same time, provide more time for merchants still using the 1.6 version to get ready to upgrade, by keeping them safe in case of critical issue [after the official support ends on the 30th of June 2019[(https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6). We have come up with a solution, thanks to the initiative and support of a few volunteers from the PrestaShop developer community: allowing them to take over the responsibility of maintaining PrestaShop 1.6.1.x beyond the end of official support.
```

Isn't it strange to have the dot in the middle of a statement about a dilemna ? 🤔

Reply:
English prefer short sentences and this has been approved by Pablo, so, I will keep that ;-)

Reply:
```suggestion
This last year, the PS company has been facing a dilemma: how to dedicate 100% of the developers’ time to move forward faster on the 1.7 version of the open source project, something everybody agrees on. And, at the same time, provide more time for merchants still using the 1.6 version to get ready to upgrade, by keeping them safe in case of critical issue [after the official support ends on the 30th of June 2019](https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6). We have come up with a solution, thanks to the initiative and support of a few volunteers from the PrestaShop developer community: allowing them to take over the responsibility of maintaining PrestaShop 1.6.1.x beyond the end of official support.
```

Bad `[` => `]`